src/ui/primitives/styles.rs

```diff
@@ -47,6 +47,10 @@ pub fn settings_line(s: Style) -> Style {
 		.gap(0, 5)
 }
 
+pub fn multiline(s: Style) -> Style {
+	s.size_full().padding(5).border_radius(2).border(1).border_color(C_TOP_TEXT)
+}
```

Comment:
I can't wrap the multiline input into a container because I need to look into `doc()` which I won't have access to if the editor is wrapped in a `container` so I had to make a style function... Not happy with the editor... lot's of issues